src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -195,6 +195,10 @@ public void setSubmitAction(String submitAction) {
         return result;
     }
     
+    public String getEncryptedRegkey() {
+        return StringHelper.encrypt(student.key);
+    }
```

Comment:
Not exactly related to this issue, but should we have a new field `encryptedKey` on StudentAttributes (although not persisted to the database)? Other than the search document's id, it doesn't actually seem like there is a valid use of `student.key` anywhere.